src/lib/tile-processing/vector/qualifiers/factories/vector-tile/helpers/getBuildingParams.ts

```diff
@@ -40,7 +40,7 @@ export default function getBuildingParams(
 
 	const roofParams = getRoofParams(tags);
 	const roofOrientation = getRoofOrientation(<string>tags['roofOrientation']);
-	const roofLevels = <number>tags.roofLevels ?? (roofParams.type === 'flat' ? 0 : 1);
+	const roofLevels = tags.roofLevels <= 0 ? 1 : <number>tags.roofLevels ?? (roofParams.type === 'flat' ? 0 : 1);
```

Comment:
Perhaps this would be more inline with the reality. 
```suggestion
	const roofLevels = tags.roofLevels <= 0 ? 0.5 : <number>tags.roofLevels ?? (roofParams.type === 'flat' ? 0 : 1);
```

Reply:
Good idea! .5 was a bit too low but .6 is good:

![image1](https://github.com/StrandedKitty/streets-gl/assets/26939824/6d78aa91-c9f3-4e00-b822-bdfbe41bcfa1)

Reply:
> Good idea! .5 was a bit too low 

You mean, too low to exact match your random example :-)

> but .6 is good:

But whatever. If the mapper tagged 0 we have to guess something for non flat roofs...